devlab.yml

```diff
@@ -0,0 +1,25 @@
+# DOCKER-VERSION 1.9.1
+# DEVLAB-VERSION 1.5.3
+
+#
+# Stardust DevLab YAML
+#
+# http://fluidbyte.net/docker/devlab/nodejs/2015/10/15/containerize-your-local-dev-in-minutes-with-devlab/
+# https://github.com/TechnologyAdvice/stardust/
+#
+
+from: node:argon
```

Comment:
Good to know! Will update before merge.
On Sun, Dec 6, 2015 at 12:51 PM Tom Shawver <notifications@github.com>
wrote:

> In devlab.yml
> <https://github.com/TechnologyAdvice/stardust/pull/125#discussion_r46774116>
> :
>
> > @@ -0,0 +1,25 @@
> > +# DOCKER-VERSION 1.9.1
> > +# DEVLAB-VERSION 1.5.3
> > +
> > +#
> > +# Stardust DevLab YAML
> > +#
> > +# http://fluidbyte.net/docker/devlab/nodejs/2015/10/15/containerize-your-local-dev-in-minutes-with-devlab/
> > +# https://github.com/TechnologyAdvice/stardust/
> > +#
> > +
> > +from: node:argon
>
> Not only for consistency-- our container is about a fifth of the size of
> the official one. They insist on building on top of a full Debian install.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/TechnologyAdvice/stardust/pull/125/files#r46774116>.
>